src/styles/_catalog.scss

```
@@ -16,7 +16,19 @@
     margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023.2px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 488px) and (max-width: 767.2px) {
```

```suggestion
  @media (min-width: 488px) and (max-width: 767px) {
```
no need decimal pixels